users/models.py

```diff
@@ -93,7 +93,8 @@ class Meta:
 
     email = models.EmailField(
         unique=True,
-        blank=False,
+        blank=True,
+        null=True,
```

Comment:
unique=true and null=true?

Can we figure out a better way for this? for example setting the email address to devnull<originalid>@tampere.hacklab.fi ?

Or at least I would like to see a new objectmanager that can be used to always only fetch "active" users. Something like CustomUser.activeUsers.all(). (Or even in such a way that objects always returns only active and have another allObjects manager that returns also the "deleted" users.

Reply:
I don't like fake empty values such as devnull suggested. Null & blank is better if the value can be blank as it now can be. We just need to make sure that it's not empty on registration and can become empty only on resignation.